generator/config/config.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+use MongoDB\Aggregation\Operator;
+use MongoDB\Query\Operator as QueryOperator;
+use MongoDB\Aggregation\Stage;
+use MongoDB\CodeGenerator\FactoryClassGenerator;
+use MongoDB\CodeGenerator\ValueClassGenerator;
+
+$src = __DIR__ . '/../../src';
+
+return [
+    // Stages
+    [
+        'configFile' => __DIR__ . '/stages.yaml',
+        'generatorClass' => ValueClassGenerator::class,
+        'namespace' => MongoDB\Aggregation\Stage::class,
```

Comment:
About Aggregation Builder namespacing. I'd like to put all the classes in the same `MongoDB\Operator` namespace. They are grouped under this name this in the documentation: https://www.mongodb.com/docs/manual/reference/operator/

```
MongoDB\Operator\Query
MongoDB\Operator\Project
MongoDB\Operator\Update
MongoDB\Operator\Stage
MongoDB\Operator\Pipeline
```
(update operator example is for future, not implemented in this project).

Each of this would be classes with static factory methods for value-object classes in their sub-namespace. So we will have:

```
MongoDB\Operator\Query\AndQuery
MongoDB\Operator\Query\EqQuery
MongoDB\Operator\Query\ElemMatchQuery
...
MongoDB\Operator\Project\ElemMatchProjection
...
MongoDB\Operator\Update\SetUpdate
MongoDB\Operator\Update\PopUpdate
...
MongoDB\Operator\Stage\BucketStage
MongoDB\Operator\Stage\ProjectStage
MongoDB\Operator\Stage\SetStage
...
MongoDB\Operator\Aggregation\AbsAggregation
MongoDB\Operator\Aggregation\BinarySizeAggregation
MongoDB\Operator\Aggregation\EqAggregation
MongoDB\Operator\Aggregation\BinarySizeAggregation
MongoDB\Operator\Aggregation\FirstNAggregation
MongoDB\Operator\Aggregation\MaxNAggregation
```

@jmikola what do you think?

Reply:
Note that we already have the `MongoDB\Operation` namespace and there could be confusion between `MongoDB\Operation\Aggregate` and `MongoDB\Operator\Aggregation`.

We can choose `MongoDB\Builder` as namespace. Like "query builder", "aggregation builder", "update builder" ...

Reply:
What is the intention of repeating the namespace component as a class name suffix? Is that to avoid requiring `use as` if multiple operators are included in the same file?

Reply:
Avoid name restrictions (`And` and `Match` are not allowed as class names) and avoid `use as` when needing to use operators with the same name in the same file (`Query\Eq` and `Aggregation\Eq`).

But this classes will not be used outside of this library or ORMs.

Reply:
If you have alternative suggestions for the main namespace `MongoDB\Builder\` and `MongoDB\Builder\Aggregation` for aggregation pipeline operators, I'm all ears.

Reply:
I think the current namespaces are fine, especially if users are unlikely to interface with these directly (assuming they use factory methods). We can always revisit later if needed.